spec/unit/pops/types/type_calculator_spec.rb

```diff
@@ -208,6 +208,12 @@ def types
       end
     end
 
+    context 'sensitive' do
+      it 'translates to PSensitiveType' do
+        expect(calculator.infer(PSensitiveType::Sensitive.new("hunter2")).class).to eq(PSensitiveType)
+      end
+    end
```

Comment:
We would expect that the type of `Sensitive("hunter")` is `Sensitive[String[6,6]]`